src/service/MilanApi.js

```diff
@@ -136,12 +136,12 @@ export const GetAllEvents = async () => {
 };
 
 //* CONTACT
-export const Contact = async (formData, toast) => {
+export const Contact = async (formData) => {
   try {
     const response = await Axios.post(Contact_Us, formData);
     toast.success(response.data.message);
```

Comment:
Can we perhaps just return the `response` as we do for the others ? 

Example: 

```javascript
 const User = await Axios.post(User_Log, credentials);
 return User;
```

That we no longer would have to use `toast.success(response.data.message);` here, instead we can head over to Contact.jsx and then can use the response.

Example: 

```javascript
 const Data = LoginUser(credentials);

      Data.then((response) => {
        if (response?.data.token) {
          Cookies.set("token", response.data.token);
          showSuccessToast("Logged you in !");
          Navigate("/");
        } else {
          setCredentials({ email: "", password: "" });
        }
      }).catch(() => {
        showErrorToast("Server error, try again later !");
      });
```

Though instead of ` if (response?.data.token)` we can use ` if (response?.status===201)`